Tasks/CondaAuthenticateV0/task.json

```diff
@@ -0,0 +1,33 @@
+{
+ "id": "543c39b2-11e9-4de8-8b9b-94eea1339891",
+ "name": "CondaAuthenticate",
+ "friendlyName": "Conda authenticate (for task runners)",
+ "description": "Authentication task for the conda client",
+ "helpUrl": "",
+ "helpMarkDown": "",
+ "category": "Package",
+ "author": "Microsoft Corporation",
+ "version": {
+     "Major": 0,
+     "Minor": 221,
```

Comment:
Change this to 220, we will deploy this task with the current sprint to expedite the rollout